src/Lykke.Logs/Obsolete/LykkeLogToAzureStorage.cs

```diff
@@ -6,9 +6,13 @@
 using Autofac;
 using Common;
 using Common.Log;
+using Lykke.Common;
+using Lykke.Common.Log;
+using Microsoft.Extensions.Logging;
 
 namespace Lykke.Logs
 {
+    [Obsolete("Use new Lykke logging system")]
```

Comment:
Can we replace this to cref of a type from new logging system?

Reply:
There is no direct replacement for this class